TechAndCheck/zenodotus
# Implement designed navbar

This PR slow-rolls our way into the new design by way of the navbar, which goes from this:
![header-old](https://user-images.githubusercontent.com/4731/194670781-dff2570f-9c49-41e0-a2b0-5e03b6f3e6b1.png)

To this:
![header-new](https://user-images.githubusercontent.com/4731/194670787-73eb4a09-58e9-4a7a-8c97-910dfc6a8224.png)

**Testing:**
- 🚨 `rails assets:precompile`
   - Something about the change to "real" domains broke live asset delivery. I suspect I'll be able to fix it when I purge Node again as part of #290.
- `rails s`
- Make sure the navbar looks like the [design](https://xd.adobe.com/view/082065b6-ada4-454e-a03c-80defb22f2e7-5d73/) (within reason)
- Log in and note the "user" menu; switch between Insights and Vault in the user menu
- Narrow the window to trigger the mobile viewport

⚠️ Note that everything below the navbar (content, footer, etc.) is either unchanged or still in flight.

Affects #267
